Tests/modules/io_related/test_codecs.py

```diff
@@ -470,10 +473,18 @@ def run_python(filename):
 
         self.assertEqual(len(t_err_lines), 0)
         if is_cli:
-            print("CodePlex 11334")
-            self.assertEqual(t_out_lines[0].rstrip(), b"\xe6ble")
+            # CodePlex 11334: IronPython uses active console codepage for output
+            # The test below assumes cp850, which is default on Western European/Northern American Windows
```

Comment:
I'm not sure that this is necessarily correct. On my system the terminals are cp437 and the test is skipped even if it's valid. I'm not sure what encoding the CI uses. Would be nice not to have something so specific.

`not is_netcoreapp` can probably be replaced with `import clr; clr.AddReference("System.Console")`?

Reply:
My mistake. I've discovered it by chance and now try to reverse-engineer the behaviour. On my system the console uses cp65001 and `PYTHONIOENCODING` is set to `utf-8:surrogateescape`. As a result, both IronPython and CPython were failing this test. My intention was to have this test skipped on my system (unless I reconfigure a console for testing) but pass on CI. Also, it should not fail on any other system that somebody may check it out on. 

What I propose: replace the constant `b"\xe6ble"` with a `'µble'` encoded according to the actual codepage in use. 

On a side note: since the console codepage turns out to be a significant enviromental factor, maybe display it in the "Version Information" section on CI? I also notice that "Version Information" section is missing in Windows CI builds.

As for CPython, things are also not that simple as my simple test condition suggests, esp. on POSIX and higher versions of Python (interplay of `LC_ALL`, `LC_CTYPE`, `PYTHONUTF8`, `PYTHONIOENCODING`, `PYTHONCOERCECLOCALE`, and maybe more). Again, my intention was to make this test runnable at least on CI and in my environment. Luckily (?...) the test is skipped on POSIX for an unrelated reason. The test may have to be revised once `Popen` on POSIX becomes available.

> `clr.AddReference("System.Console")`

Good idea.

Reply:
Yes, maybe something like:
```Python
self.assertEqual(t_out_lines[0].rstrip(), "\xb5ble".encode(System.Console.OutputEncoding))
```